sql/catalyst/src/main/scala/org/apache/spark/sql/util/ArrowUtils.scala

```diff
@@ -45,6 +46,7 @@ object ArrowUtils {
     case BinaryType => ArrowType.Binary.INSTANCE
     case DecimalType.Fixed(precision, scale) => new ArrowType.Decimal(precision, scale)
     case DateType => new ArrowType.Date(DateUnit.DAY)
+    case MapType(k, v, nullable) => new ArrowType.Map(false)
```

Comment:
Do we need this?
Seems like complex types are handled in `toArrowField` and `fromArrowField`?

Reply:
Fixed